tests/chaos/oadp/test_oadp.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+import pytest
+
+from utilities.constants import TIMEOUT_10MIN, Images
+
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.mark.chaos
+@pytest.mark.volume_mode("block")
```

Comment:
Why only block? Can't this test run on a Filesystem storage?

Reply:
Deleted